src/app/challenges/[id]/HintBox.tsx

```diff
@@ -38,7 +38,7 @@ export default function HintBox({ challenge }: { challenge: Challenge }) {
       )}
       <h3 className="flex grow justify-center px-3 text-center">
         {hintIndex === undefined
-          ? <p>Do you need a hint?</p>
+          ? <p>Do you need a hint? <br/> Tap the arrow!</p>
```

Comment:
I've tried this with a screen reader (meta, I know, but the accessibility challenge should be accessible), and the screen reader would read that button as "greater than", so this message wouldn't help them. Let's chat about it tomorrow

Reply:
I've added an 'aria-label' to each but haven't tested so you will probably still want to take another look at this one.

Reply:
TalkBack still does not like this, I'll have a look tomorrow